### DISCUSSION TITLE
useAsyncData documentation

### DISCUSSION THREAD

**Discussion by YuLogun** [nuxt/nuxt#16773] [Questions]
It states the following [here](https://v3.nuxtjs.org/api/composables/use-async-data/#return-values):
**By default, Nuxt waits until a refresh is finished before it can be executed again. Passing `true` as parameter skips that wait.**

I don't really understand this rule. Where should I pass `true` if I want to skip the wait?

My use case is a search-as-you-type functionality similar to this issue https://github.com/nuxt/framework/issues/2973.
But now if a request is already pending, a new one won't fire.

**Comment by sevillaarvin** [comment]
https://github.com/nuxt/framework/blob/a9a8571e352da37fae957967c6dd43ac67b58da0/packages/nuxt/src/app/composables/asyncData.ts#L123

I think the public apis are still subject to change, but currently you need to pass `{ _initial: false }` to force execute.
